src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -538,6 +558,20 @@ Result Device::CreateApiObjects(const grfx::DeviceCreateInfo* pCreateInfo)
     }
 #endif
 
+    PPX_LOG_INFO("Vulkan MultiView is chosen and present: " << mHasMultiView);
+    if (mHasMultiView) {
+        VkPhysicalDeviceMultiviewFeatures physicalDeviceMultiviewFeatures =
+            {
+                VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_MULTIVIEW_FEATURES,
+                nullptr,
+                VK_TRUE,  // multiview
+                VK_FALSE, // multiviewGeometryShader
+                VK_FALSE  // multiviewTessellationShader
+            };
+
+        extensionStructs.push_back(reinterpret_cast<VkBaseOutStructure*>(&physicalDeviceMultiviewFeatures));
```

Comment:
This is adding a pointer to a variable that will be out of scope when it is used later. `physicalDeviceMultiviewFeatures` should be defined above the if statement